node/_process/streams.mjs

```diff
@@ -101,9 +101,36 @@ export const stdout = stdio.stdout = createWritableStdioStream(
   "stdout",
 );
 
+function _adjustHighWaterMark() {
+  if (Deno.isatty?.(Deno.stdin?.rid)) return 0; // stdin is a TTY
+
+  if (Deno.build.os !== "windows") {
+    const fileInfo = Deno.fstatSync?.(Deno.stdin?.rid);
+    if (fileInfo.isFile) return 64 * 1024; // stdin is a redirected file
```

Comment:
I see this constant appearing multiple times in this file, it seems like it could be beneficial to break it out into a constant or enum and reference that instead. A descriptive name could help a lot.

I'm not totally sure what each value signifies so I'm guessing below, but you get the idea.

```ts
enum HighWaterMark {
  Interactive = 0,
  Ignore = 64 * 1000,
}
```
```ts
if (fileInfo.isFile) return HighWaterMark.Ignore;
```

Reply:
Thanks for the advice.
I will do some code refactoring to take into account the feedback I have received before changing this PR from Draft to Open.